### DISCUSSION TITLE
ManagementObject set IPv6?

### DISCUSSION THREAD

**Discussion by CodingOctocat** [dotnet/runtime#94552] [Q&A]
Why does `ManagementObject` not support set IPv6?

**Comment by danmoseley** [comment]
System.Management is a legacy wrapper of an older Windows management technology. It is ported to .NET Core only to enable ports of code from .NET Framework and in general it is not a goal to add features to it but rather to keep it stable. That being said, can you explain your use case?

**Reply by CodingOctocat** [reply]
I've written a GUI program for IP configuration, which doesn't support IPv6 yet, and I know I can call `netsh`, But I want to keep the code uniform.

**Reply by danmoseley** [reply]
@daxian-dbw I'm not familiar with what modern Windows management tech there is other than Powershell. What's the recommendation for someone who wants to do management tasks from C#?
